package.json

```diff
@@ -28,17 +28,18 @@
     "@docsearch/react": "^3.6.1",
     "@headlessui/react": "^1.7.0",
     "@radix-ui/react-context-menu": "^2.1.5",
+    "babel-plugin-react-compiler": "0.0.0-experimental-696af53-20240625",
```

Comment:
@mattcarrollcode this is a super old version of the compiler, I'd suggest using one of the beta releases

Reply:
#7428 7428